x/staking/module.go

```diff
@@ -30,12 +33,12 @@ func NewAppModule(cdc codec.Codec, keeper *keeper.Keeper, ak stakingtypes.Accoun
 }
 
```

Comment:
Add to two interface
```
_ appmodule.HasBeginBlocker = AppModule{}
_ module.HasABCIEndBlock    = AppModule{}
```

Reply:
https://github.com/xpladev/xpla/pull/154/commits/76a6661211f895f81c2c7a9dd4ff25a874613080